<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkIGIDataSourceManager</class>
 <widget class="QWidget" name="QmitkIGIDataSourceManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>838</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkIGIDataSourcemanager</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="m_SourceSelectComboBox"/>
       </item>
       <item>
        <widget class="QSpinBox" name="m_PortNumberSpinBox">
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>3200</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_AddSourcePushButton">
         <property name="maximumSize">
          <size>
           <width>22</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_RemoveSourcePushButton">
         <property name="maximumSize">
          <size>
           <width>22</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="m_HorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="ctkPathLineEdit" name="m_FileChooser" native="true"/>
       </item>
       <item>
        <widget class="QPushButton" name="m_PlayPushButton">
         <property name="toolTip">
          <string>play back data from a previously recorded session</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_RecordPushButton">
         <property name="toolTip">
          <string>record (save) data</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_StopPushButton">
         <property name="toolTip">
          <string>stop recording or playing</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ctkCollapsibleGroupBox" name="m_ToolManagerToolsGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Sources</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableWidget" name="m_TableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>6</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>device</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>description</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>rate</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>lag</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="ctkCollapsibleGroupBox" name="m_ToolManagerConsoleGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Console</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPlainTextEdit" name="m_ToolManagerConsole">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="ctkCollapsibleGroupBox" name="m_ToolOutputGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Output</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="m_Frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
